### DISCUSSION TITLE
Zigbee dongle speed

### DISCUSSION THREAD

**Discussion by lgLindstrom** [Koenkk/zigbee2mqtt#12176] [General]
Hi

When struggling with my problems regarding my zigbee dongel I noticed that the communication speed between the electronics in dongle and the computer is 115200 bits/sec.

One of the problems in my "smart home" is the delay between, for example, wife presses a button,,, and the bulb changes state. 

I understand that there are many steps involved in this "action" but will a faster communication speed make any notable difference and is there any zigbee dongles that are faster?

**Comment by lux73** [comment]
don't know which Coordinator you have but my CC2652RB from[ slae.sh](https://slae.sh/projects/cc2652/) in combination with z2m & OpenHAB didn't have any noticeable Delay

i'm using many Motion Sensors form Xiaomi in nearly all Rooms - all of them switch the Lights without any Delay. Even my Xiaomi Double Rockers haven't any Delay when switching Lights

**make sure you have an USB extension Cable between your USB Port & the Coordinator.** If i not using any extension & sticking my Coordinator directly in the USB Port i have terribly Delay and dropouts...

**Reply by Kimovitzh** [reply]
@lux73 are you using a raspberry pi? And how log is the extension?

**Reply by lux73** [reply]
no, i'm using [apu2c4](https://www.pcengines.ch/apu2c4.htm) Board - my USB Extension Cable is 2m long. Any shorter will even work
